pydevices/acq400Devices/acq1001_st.py

```diff
@@ -24,22 +24,27 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
-try:
-    acq400_base = __import__('acq400_base', globals(), level=1)
-except:
-    acq400_base = __import__('acq400_base', globals())
+import importlib
 
+acq400_base = importlib.import_module('acq400_base')
+acq400_hapi = importlib.import_module('acq400_hapi')
 
 class _ACQ1001_ST(acq400_base._ACQ400_ST_BASE):
     """
     D-Tacq ACQ2106 stream support.
     """
+    def init(self):
+        super(_ACQ1001_ST, self).init()
+        uut = acq400_hapi.Acq400(self.node.data(), monitor=False)
+
+        uut.s0.set_knob('SYS_CLK_FPMUX', 'XCLK')
```

Comment:
The parent class is ACQ400_ST_BASE, which inherits 'init' from ACQ400_BASE. Creating a new init here would replace that init function. This init doesn't do much anyway, better removed I think.